test.md

```diff
@@ -0,0 +1 @@
+"this file should be ignored by git"
```

Comment:
@iam-nadeem this file should be ignored as per the requirements which means the test.md file should not be there in the repo 👍